src/hooks/useApplicationsInfo.ts

```diff
@@ -18,7 +18,10 @@ export const useApplicationsInfo = (): ApplicationInfo => {
   React.useEffect(() => {
     let mounted = true;
     const fetchApplication = async () => {
-      const appData = await k8sGetResource({ model: ApplicationModel, ns: namespace });
+      const appData: any = await k8sGetResource({
```

Comment:
@rohitkrai03 Do you guys have K8sResourceCommon-esk types?

Reply:
@rohitkrai03 You will get common types for sure... but use-case specific types is beyond the scope of the SDK. You'll need to base your types off a common one in your world for your own use-cases. If we supported all types than the SDK would need to be the source of truth for several projects/plugins and it becomes a bit unmanageable and violates the separation of concerns principle.

I do believe we have an issue with exporting the `K8sResourceCommon` type right now, but it should be there.